specification/loadtestservice/LoadTestService/tspconfig.yaml

```diff
@@ -1,6 +1,6 @@
 parameters:
-  service-directory-name:
-    default: "loadtesting"
+  service-dir:
+    default: "sdk/loadtesting"
```

Comment:
The SDK part should be part of your repo path

Reply:
I may not get your point completely and I referred the example path here: https://github.com/Azure/azure-rest-api-specs/blob/dedb2ee1024fbda335fe8742b8a6f3603474af3a/specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml#L3.

cc @raych1

Reply:
@catalinaperalta , this is default value, the package is dropped under `sdk` folder in each language SDK repos.

Reply:
Why was this change made? The `sdk` part is common in all of the repositories, seems like this shouldn't be part of the configuration section

Reply:
@raych1 Could you help on this?

Reply:
@catalinaperalta , this is the [PR](https://github.com/Azure/azure-rest-api-specs/pull/23866) of updating the `tspconfig.yaml` schema which includes the details on the background FYI.